internal/adapters/cloudformation/aws/rds/cluster.go

```diff
@@ -9,56 +9,26 @@ import (
 func getClusters(ctx parser.FileContext) (clusters map[string]rds.Cluster) {
 	clusters = make(map[string]rds.Cluster)
 	for _, clusterResource := range ctx.GetResourcesByType("AWS::RDS::DBCluster") {
-		cluster := rds.Cluster{
+		clusters[clusterResource.ID()] = rds.Cluster{
 			Metadata:                  clusterResource.Metadata(),
-			BackupRetentionPeriodDays: defsecTypes.IntDefault(1, clusterResource.Metadata()),
-			ReplicationSourceARN:      defsecTypes.StringDefault("", clusterResource.Metadata()),
+			BackupRetentionPeriodDays: clusterResource.GetIntProperty("BackupRetentionPeriod", 1),
+			ReplicationSourceARN:      clusterResource.GetStringProperty("SourceDBInstanceIdentifier"),
 			PerformanceInsights: rds.PerformanceInsights{
 				Metadata: clusterResource.Metadata(),
-				Enabled:  defsecTypes.BoolDefault(false, clusterResource.Metadata()),
-				KMSKeyID: defsecTypes.StringDefault("", clusterResource.Metadata()),
+				Enabled:  clusterResource.GetBoolProperty("PerformanceInsightsEnabled"),
+				KMSKeyID: clusterResource.GetStringProperty("PerformanceInsightsKmsKeyId"),
```

Comment:
should this be the following?

```suggestion
				KMSKeyID: clusterResource.GetStringProperty("PerformanceInsightsKMSKeyId"),
```

Reply:
Either way, we should add it to the test case so we can cover it.

Reply:
Must be [PerformanceInsightsKmsKeyId](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-performanceinsightskmskeyid)

Reply:
[Done](https://github.com/aquasecurity/trivy-iac/pull/69/commits/f7d88abf8367ee594a48ebce787a120eaea34017#diff-613168e6eaf30167b7831f0744245b71bbcf03248176424b6be56bb3eb2d4282)